src/cpu/aarch64/acl_post_ops.hpp

```diff
@@ -0,0 +1,225 @@
+                    eltwise_desc.primitive_kind = primitive_kind::eltwise;
+                    eltwise_desc.alg_kind = po.eltwise.alg;
+                    eltwise_desc.data_desc = dst_md;
+                    eltwise_desc.prop_kind = prop_kind_t::dnnl_forward;
+                    auto empty_attr = dnnl_primitive_attr();
+                    auto acl_eltwise_pd = std::make_shared<
+                            typename acl_eltwise_fwd_t::pd_t>(
+                            &eltwise_desc, &empty_attr, nullptr);
+                    CHECK(acl_eltwise_pd->init(engine));
+                    post_op_pds.push_back(acl_eltwise_pd);
+                } else {
+                    // Unsupported catchall
+                    return status::unimplemented;
+                }
+            }
+            return status::success;
+        }
+
+        // init by copying post ops, note that this function modifies the passed
+        // in post ops by setting the preferred memory formats
+        status_t init(engine_t *engine, const memory_desc_t &dst_md,
+                post_ops_t &post_ops) {
+            CHECK(post_ops.set_default_formats(&dst_md));
+            CHECK(post_ops_.copy_from(post_ops));
+            return init(engine, dst_md);
+        }
+
+        // init by popping off the first post op if it is eltwise (to be fused
+        // into the base primitive) and then copying the rest. Note that this
+        // function modifies the passed in post ops by setting the preferred
+        // memory formats
+        status_t init(engine_t *engine, const memory_desc_t &dst_md,
+                post_ops_t &post_ops,
+                arm_compute::ActivationLayerInfo &act_info_to_fuse) {
+
+            CHECK(post_ops.set_default_formats(&dst_md));
+
+            // If the first entry is eltwsie, we fuse it
+            if (post_ops.len() >= 1 && post_ops.entry_[0].is_eltwise()) {
+
+                CHECK(acl_utils::convert_to_acl_act(
+                        post_ops.entry_[0].eltwise, act_info_to_fuse));
+
+                // Copy all but the first, because it has been fused
+                post_ops_.entry_.clear();
+                for (int idx = 1; idx < post_ops.len(); ++idx) {
+                    // Construct empty entry then copy, so that we can check for failure
+                    post_ops_.entry_.emplace_back();
+                    CHECK(post_ops_.entry_.back().copy_from(
+                            post_ops.entry_[idx]));
+                }
+            } else {
+                // Nothing to fuse, just copy all post ops
+                CHECK(post_ops_.copy_from(post_ops));
+            }
+
+            return init(engine, dst_md);
+        }
+
+        bool has_sum() const { return acp_.sum_index >= 0; }
+
+        acl_post_ops_conf_t acp_;
+
+        // Vector of acl_binary_t::pd_t or acl_eltwise_fwd_t::pd_t to construct
+        // primitives for later
+        std::vector<std::shared_ptr<primitive_desc_t>> post_op_pds;
```

Comment:
(minor): Usually we use `_` at the end of members:
```suggestion
        std::vector<std::shared_ptr<primitive_desc_t>> post_op_pds_;
```

Reply:
Does this apply if they are public?

Reply:
Yes. Just to clarify, it was a minor suggestion, so you can close it without fixing. I see that other classes use `mtx` and not `mtx_`, so i am fine keeping it as is.

Reply:
No problem at all, I'd much rather go with the convention. It was just that I had noticed that a lot of the public members (in some of the lower level structs like the pds and descs) had no underscore, and thought that was the pattern.

Reply:
O, now I see what you meant by "public". Public API doesn't contain `_`. `_` is used in internal structures only. Actually I don't see this requirement in [Code standards](https://github.com/oneapi-src/oneDNN/blob/master/CODING_STANDARDS.md) so please ignore the suggested change. Sorry for noise.

Reply:
No worries, thanks for the clarification!